README.md

```diff
@@ -12,66 +12,88 @@ And I guess I'm just yet another one looking at the xkcd:File Transfer thing:
+Attempting NAT traversal:
+ < < < < < > > > > >
+NAT traversal pass succeeded
+0 - Trying to connect to 176.xx.xx.xx:62701.
+priv.state==TCP_SYN_SENT
+Waiting for TCP_ESTABLISHED...
+priv.state==TCP_ESTABLISHED
+Connected successfully to 176.xx.xx.xx:62701
+Called PeerConnect
+\[###-------------------------------]       585.92 KB / 5 MB.  117.18 KB/s
 ```
 
 ## Receiver
 ```
->p2pcopy.exe receiver --localport 21300
+$ mono p2pcopy.exe --localport 21300 receiver
 Using local port: 21300
 Your firewall is FullCone
-Tell this to your peer: 188.44.136.7:21300
-
+p2pEndPoint external=176.xx.xx.xx:62701
+p2pEndPoint internal=0.0.0.0:21300
+Tell this to your peer: 176.xx.xx.xx:62701
 
-Enter the ip:port of your peer: 223.154.44.121:4300
+Enter the ip:port of your peer: 18.216.90.92:4300
 Your firewall is FullCone
-[5:51:56 PM] - Waiting 4 sec to sync with other peer
-Your firewall is FullCone
-0 - Trying to connect to 223.154.44.121:4300.  Connected successfully to 223.154.44.121:4300
--[##########################################---------------]      86 MB / 181.64 MB.     688 KB/s
-
+Getting internet time
+[12:33:37 PM] - Waiting 8 sec to sync with other peer
+After reusing existing sock:
+UdpClient.Client.LocalEndPoint=192.168.1.217:21300
+UdpClient.Client.RemoteEndPoint=18.216.90.92:4300
+underlyingSock.LocalEndPoint=192.168.1.217:21300
+Attempting NAT traversal:
+ > > > > > < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < <
+NAT traversal pass succeeded
+0 - Trying to connect to 18.216.90.92:4300.
+priv.state==TCP_LISTEN
+Waiting for TCP_ESTABLISHED...
+priv.state==TCP_SYN_RECEIVED
+Waiting for TCP_ESTABLISHED...
+priv.state==TCP_ESTABLISHED
+Connected successfully to 18.216.90.92:4300
+Called PeerConnect
+Receiving file: 5MB.zip
+File size=5242880 bytes
+\[#####-----------------------------------------------]       495.92 KB / 5 MB.   99.18 KB/s
 ```
 
 ## Potential connection problems
-Sometimes the two peers try to punch a hole on their routers but they don't succeed. If that happens, simply retry and eventually (normally works well) it will work.
-
-A sample failed session looks as follows:
-```
->p2pcopy.exe sender --localport 60300 --file 03183u.tif
-Using local port: 60300
-Your firewall is FullCone
-Tell this to your peer: 2xx.2xx.84.121:60300
+Sometimes the two peers try to punch a hole on their routers but they don't succeed. If that happens, simply leave the tool running and eventually (normally works well) it will work.
```

Comment:
what does `simply leave the tool running` mean? why are you removing the "retry"?

Reply:
Because the two sides sync quite well in my experience without needing a restart of the tool. However it's possible the hole in the firewall would timeout, so perhaps we should retain the "retry". Will change back.

Reply:
Have added back "retry" in latest README.